controllers/policyserver_controller_test.go

```diff
@@ -559,6 +560,79 @@ var _ = Describe("PolicyServer controller", func() {
 			}).Should(Succeed())
 		})
 
+		It("the deployment must have the configMap version as an annotation", func() {
+			policyServer := policyServerFactory(policyServerName)
+			createPolicyServerAndWaitForItsService(policyServer)
+			Eventually(func() error {
+				configmap, err := getTestPolicyServerConfigMap(policyServerName)
+				if err != nil {
+					return err
+				}
+				deployment, err := getTestPolicyServerDeployment(policyServerName)
+				if err != nil {
+					return err
+				}
+				if deployment.GetAnnotations()[constants.PolicyServerDeploymentConfigVersionAnnotation] != configmap.GetResourceVersion() {
+					return errors.New("deployment configmap version did not change")
+				}
+				if deployment.Spec.Template.GetLabels()[constants.PolicyServerDeploymentConfigVersionAnnotation] != configmap.GetResourceVersion() {
+					return errors.New("pod configmap version did not change")
+				}
+				return nil
+			}, timeout, pollInterval).Should(Succeed())
+		})
+
+		It("the deployment must have the configMap version update after adding a policy to the policy server", func() {
+			policyServer := policyServerFactory(policyServerName)
+			createPolicyServerAndWaitForItsService(policyServer)
+			Eventually(func() error {
+				configmap, err := getTestPolicyServerConfigMap(policyServerName)
+				if err != nil {
+					return err
+				}
```

Comment:
move outside, and call it `initialConfigMap` or something like that. You can use it later on instead of `oldConfigMap`